docs/plugins.md

```diff
@@ -38,6 +38,12 @@ If the plugin is unable to be found automatically, you can load them with:
 * [`@prettier/plugin-php`](https://github.com/prettier/prettier-php)
 * [`@prettier/plugin-swift`](https://github.com/prettier/prettier-swift)
 
+## Unofficial Plugins
```

Comment:
Other ideas for the title are more than welcome

Reply:
"Community Plugins" might be a bit more friendly

Reply:
Much better!